packages/stylelint-config/scss.js

```diff
@@ -0,0 +1,36 @@
+'use strict';
+
+module.exports = {
+	extends: [ './', 'stylelint-config-recommended-scss' ].map(
+		require.resolve
+	),
+
+	plugins: [ 'stylelint-scss' ],
+
+	rules: {
+		// stylelint-config-wordpress css overrides
```

Comment:
Should it get updates as well?
```suggestion
		// @wordpress/stylelint-config css overrides
```